### PR TITLE
docs(connectors): explained several fields

### PR DESCRIPTION
**Pull request by igpetrov**
## Description

Some minor adjustments

## When should this change go live?

Anytime

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [x] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [x] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by igpetrov** [comment]
Hi @christinaausley. This one is ready to be merged after review.

Note: as a part of experiment, I asked AI to rewrite my own writings at `docs/components/connectors/use-connectors/index.md`. It looks good to me but you, as a native speaker, please take a look whether AI outcome even makes sense 😄

File: docs/components/connectors/use-connectors/index.md [hunk 1]
```diff
@@ -48,22 +48,54 @@ For further details on how secrets are implemented in Connectors, consult our [C
 Using this in other areas can lead to unexpected results and incidents.
 :::
 
-## Response mapping
+## Variable/response mapping
 
-Most Connectors have a `Response Mapping` section that typically consists of two fields: `Result Variable` and `Result Expression`. These fields are used to export responses from an external Connector call into process variables.
+In most instances, **Connectors** feature a dedicated section known as `Response Mapping`,
```
Comment (christinaausley) [inline docs/components/connectors/use-connectors/index.md:53]: In which instances will they not feature this section?

File: docs/components/connectors/use-connectors/index.md [hunk 2]
```diff
@@ -48,22 +48,54 @@ For further details on how secrets are implemented in Connectors, consult our [C
...
-This field declares a single process variable to export responses from a Connector call. You are able to use this process variable further in the process.
+This field declares a singular process variable designated to export responses from a **Connector** call.
+The resulting process variable can then be utilized within the ongoing process.
 
-### Result Expression
+### Result expression
 
-This field allows you to map a Connector response into multiple process variables which you are able to use further in the process. You can also transform the extracted values using [FEEL expressions](/components/concepts/expressions.md).
+This field facilitates the mapping of a **Connector** response into multiple process variables,
+providing further flexibility in their utilization within the ongoing process.
```
Comment (christinaausley) [inline docs/components/connectors/use-connectors/index.md:65]: "providing further flexibility in their utilization within the ongoing process" -- what is "their" here?

File: docs/components/connectors/use-connectors/index.md [hunk 3]
```diff
@@ -48,22 +48,54 @@ For further details on how secrets are implemented in Connectors, consult our [C
...
+Additionally, the extracted values can undergo transformation through the application of [FEEL expressions](/components/concepts/expressions.md).
```
Comment (christinaausley) [inline docs/components/connectors/use-connectors/index.md:66]: "undergo transformation" -- what does this mean exactly?

File: docs/components/connectors/use-connectors/index.md [hunk 4]
```diff
@@ -48,22 +48,54 @@ For further details on how secrets are implemented in Connectors, consult our [C
...
+
+To ensure process isolation, note that Connectors do not have access to process instance variables.
 
 :::note
 While using this field, a process variable with the name `response` is reserved.
 :::
 
+## Activation
+
+The `Activation` section pertains specifically to inbound **Connectors**.
```
Comment (christinaausley) [inline docs/components/connectors/use-connectors/index.md:76]: Could link to doc on types of connectors here 👍

**Review by christinaausley — COMMENTED** [review]
Fun work with AI! 🥳 

I like that AI can help kickstart so much documentation, but sometimes it uses vague terminology, hence a few comments for clarity in my review.

File: docs/components/connectors/use-connectors/index.md (hunk 1 above)
Comment (igpetrov) [reply docs/components/connectors/use-connectors/index.md]: Added

File: docs/components/connectors/use-connectors/index.md (hunk 2 above)
Comment (igpetrov) [reply docs/components/connectors/use-connectors/index.md]: Added

File: docs/components/connectors/use-connectors/index.md (hunk 3 above)
Comment (igpetrov) [reply docs/components/connectors/use-connectors/index.md]: Added

File: docs/components/connectors/use-connectors/index.md (hunk 4 above)
Comment (igpetrov) [reply docs/components/connectors/use-connectors/index.md]: Added

**Comment by igpetrov** [comment]
Hi @christinaausley 

Thank you for reviewing. Updated a PR with addressed PR comments. This one is ready to be merged at any point 🚀

**Review by christinaausley — DISMISSED** [review]

**Review by christinaausley — APPROVED** [review]

**Pull request merged**